assets/js/modules/analytics-4/components/common/WebDataStreamSelect.js

```diff
@@ -75,38 +75,9 @@ export default function WebDataStreamSelect( props ) {
 			return false;
 		}
 
-		const loadedAccounts =
```

Comment:
As per `PropertySelect`, this loading logic is extracted into a reusable `isLoadingWebDataStreams()` selector.

Reply:
Nice 👍🏻